src/index.js

```diff
@@ -1,48 +1,49 @@
-/* eslint-disable no-unused-vars */
-import _ from 'lodash';
 import './style.css';
+import {
+  getLocalStorage,
+  addToDoTask,
+  deleteToDoTask,
+  editToDoTask,
+} from './addremove.js';
+
+getLocalStorage();
+const inputTask = document.querySelector('.input-task');
+const toDoList = document.querySelector('.to-do-list');
+
+document.querySelector('.add-task').addEventListener('click', () => {
+  if (inputTask.value !== '') {
+    addToDoTask(inputTask.value);
+    inputTask.value = '';
+  }
+});
 
-const toDoList = [
-  {
-    description: 'Wash the Dishes',
-    completed: true,
-    index: 0,
-  },
-
-  {
-    description: 'Complete To Do List',
-    completed: false,
-    index: 1,
-  },
-
-  {
-    description: 'Coffee And Code',
-    completed: true,
-    index: 2,
-  },
-];
-
-window.addEventListener('load', () => {
-  const List = document.querySelector('.to-do-list');
-
-  const createElement = (e) => {
-    const createdElement = document.createElement('div');
-    const checkBox = document.createElement('input');
-    const task = document.createElement('p');
-    const hr = document.createElement('hr');
-
-    createdElement.classList.add('dynamic-Elements');
-
-    checkBox.type = 'checkbox';
-    checkBox.checked = e.completed;
```

Comment:
- [ ] Please make sure that the checkboxes are working right now they are not so like the previous reviewer asked you should make them work.